src/main/java/com/redhat/devtools/intellij/common/gettingstarted/GettingStartedLesson.java

```diff
@@ -0,0 +1,51 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc.
+ ******************************************************************************/
+package com.redhat.devtools.intellij.common.gettingstarted;
+
+import javax.swing.Action;
+import java.net.URL;
+import java.util.Collections;
+import java.util.List;
+
+public class GettingStartedLesson {
+
+    private final String title;
+    private final String description;
+    private final List<Action> actions;
+    private final URL gif;
```

Comment:
That name is not clear to me: does it needs to be a GIF image or do you use it because of animation ?. Could be better renamed as image or animatedImage ?